src/conversions/avx/rgb_xyz_opt.rs

```diff
@@ -282,7 +283,9 @@ where
                 v = _mm_min_ps(v, _mm256_castps256_ps128(v_scale));
 
                 let zx = _mm_cvtps_epi32(v);
-                _mm_store_si128(temporary0.0.as_mut_ptr() as *mut _, zx);
+                let temporary_first_half: &mut [u16; 8] =
+                    (&mut temporary0.0[..8]).try_into().unwrap();
+                _mm_storeu_si128(temporary_first_half, zx);
```

Comment:
```suggestion
                _mm_storeu_si128(temporary0.0.first_chunk_mut::<8>().unwrap(), zx);
```
This should work and avoid any chance of creating a temporary.